src/api/app/assets/stylesheets/webui/bs5-shims.scss

```diff
@@ -0,0 +1,16 @@
+@each $breakpoint in map-keys($grid-breakpoints) {
```

Comment:
How did you come up with this Sass code? It looks like something from upstream which was slightly adapted. I'm asking to understand and also to compare with how Bootstrap does this.